reactivesocket-tck-drivers/src/main/java/io/reactivesocket/tckdrivers/common/AddThread.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2016 Facebook, Inc.
+ * <p>
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ *  the License. You may obtain a copy of the License at
+ *  <p>
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  <p>
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ *  an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ *  specific language governing permissions and limitations under the License.
+ */
+
+package io.reactivesocket.tckdrivers.common;
+
+import java.util.concurrent.CountDownLatch;
+
+/**
+ * A thread that is created to wait to be able to add a marble string. We wait for the previous thread to have finished
```

Comment:
This part is copy-pasta from old code.  "marble string" refers to a marble representation (like - rr%%a%%b%%------------x--------------|&&{"x":{"hello":"goodbye"}}).  This class is used for channel tests, which is not complete as part of this PR.  But in general, the java tck-driver creates a thread per test (which seems to follow the recommendation from https://github.com/ReactiveSocket/reactivesocket-tck/blob/master/README.md).